qa/L0_infer_reshape/test.sh

```diff
@@ -44,7 +44,13 @@ for i in \
         zero_1_float32 \
         zero_2_float32 \
         zero_3_float32 \
-        zero_4_float32 ; do
+        zero_4_float32 \
```

Comment:
Did you add a passing test for Boris' exact case of 1 or more pre-pended '1' dimensions?

Reply:
Yes, the `zero_2_int32` and `zero_3_int32` models are set up for that case